chpl/chpl-service/src/main/java/gov/healthit/chpl/auth/authentication/CognitoJwtUserConverter.java

```diff
@@ -23,13 +23,14 @@ public class CognitoJwtUserConverter {
     private String region;
     private String userPoolId;
     private String clientId;
-
+    private String tokenizezRsaKeyUrl;
```

Comment:
Is this the right word here or is it a typo? Genuine question. I feel like there is an extra "z" or is it spelled that way on purpose like we might do with "clazz" ?